samples/99_zil_interpreter/app/builtins.rb

```diff
@@ -0,0 +1,21 @@
+class FunctionError < StandardError; end
+
+def define_for_evaled_args(&implementation)
```

Comment:
Given the significance of `args` in DRGTK, does it make sense to call the ZIL/MDL argument list `arglist` instead of `args`?